### PR TITLE
Do not use deprecated getDevicesByInstance method

### PR DESCRIPTION
**Pull request by vojtechtrefny**
getDevicesByInstance has been deprecated in blivet 1.7

### CODE REVIEW

File: pyanaconda/bootloader.py [hunk 1]
```diff
@@ -800,7 +800,8 @@ def set_boot_args(self, *args, **kwargs):
         if usr_device:
             dracut_devices.extend([usr_device])
 
-        netdevs = storage.devicetree.getDevicesByInstance(NetworkStorageDevice)
+        netdevs = (d for d in storage.devices if (getattr(d, "complete", True) and
+            isinstance(d, NetworkStorageDevice)))
```
Comment (vpodzime) [inline pyanaconda/bootloader.py]: This should be a list comprehension because ``netdevs`` is used at least twice in the block below.

**Comment by vpodzime** [comment]
Looks good to me otherwise. I like that this makes things more explicit and visible.

File: pyanaconda/bootloader.py (hunk 1 above)
Comment (vojtechtrefny) [inline pyanaconda/bootloader.py]: Fixed, thanks.

**Comment by vojtechtrefny** [comment]
Pushed.

**Pull request closed without merging**